app/utils/EmacUrlBuilder.scala

```diff
@@ -25,8 +25,12 @@ import play.api.mvc.Request
 @Singleton
 class EmacUrlBuilder@Inject()(appConfig: FrontendAppConfig) {
 
-  def getRequestAccessUrl(enrolmentKey: String)(vatEnrolment: Option[VatEnrolment])(implicit request: Request[_]): String =
+  def getRequestAccessEnrolUrl(enrolmentKey: String)(vatEnrolment: Option[VatEnrolment])(implicit request: Request[_]): String =
```

Comment:
The naming of these two methods is a little off. You seem to be mixing a description of the URL and the context it is used in, by this I mean the URL is `.../request-access-scheme/...` and it is used when the user is enrolling, so `getRequestAccessUrl` would be a good descriptive name of the URL and `getEnrolmentUrl` would be a good description name of when it is used. I would suggest sticking to one or the other, it makes it cleaner and clearer. As the app config calls it `emacVatEnrolmentUrl` I would suggest going with `getEnrolmentUrl`. The same applies to the function below